new/LIT7031HymnJesus.xml

```diff
@@ -0,0 +1,68 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7031HymnJesus" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Hymn to Jesus Christ</title>
```

Comment:
I took the first verse of the chant as Ethiopic title, but I was in problems in translating it. I understand the -nu in the end as question mark. ʿalāmāt, I understand as eternities (eternity before and eternity after the current world). I do not understand, why the verse uses ba-ki-nu. I understand, that -ki is a possessive suffix of the 2nd person female. In that case it could relate to Mary. But Strelcyn related the hymn to Jesus and in the other verses, a male person is adressed. The last word could also be related to bakka 'vanish/fade away' or to bakk 'vain/nothingness', but here again the vowel -i is not transparent to me. At last also the ending in -e in ʾamṣāʾe is curious to me.

Reply:
Perhaps also @nafisa-valieva likes to have a look on it?

Reply:
I would say:

አብ፡ ቅዱስ፡ አምጻኤ፡ ዓለማት፡ በኪኑ።
ʾAb qəddus ʾamṣāʾe ʿālamāt ba-kinu
The Holy Father, who brings worlds (into existence) in his skill

It's a bit tricky to translate the nomen agentis ʾamṣāʾi in a good way, so maybe a relative clause that conveys the same meaning can work

Reply:
> Perhaps also @nafisa-valieva likes to have a look on it?

I can't unsee what Jonas suggests below and for the moment it looks reasonable! :-)